example/spec/ingestGranule/IngestGranuleFailureSpec.js

```diff
@@ -27,24 +32,31 @@ const s3data = [
 ];
 
 describe('The Ingest Granule failure workflow', () => {
-  const testDataFolder = timestampedTestDataPrefix(`${config.stackName}-IngestGranuleFailure`);
+  const testId = createTimestampedTestId(config.stackName, 'IngestGranuleFailure');
+  const testSuffix = `_${testId}`;
+  const testDataFolder = createTestDataPath(testId);
   const inputPayloadFilename = './spec/ingestGranule/IngestGranule.input.payload.json';
-  const collection = { name: 'MOD09GQ', version: '006' };
-  const provider = { id: 's3_provider' };
+  const providersDir = './data/providers/s3/';
+  const collectionsDir = './data/collections/s3_MOD09GQ_006';
+  const collection = { name: `MOD09GQ${testSuffix}`, version: '006' };
```

Comment:
The collection configuration is saved in s3 as <dataType>___<version>, so changing the collection name doesn't create a new collection in the system.
Also I noticed in my test, if we use a different collection other than MOD09GQ___006, the CMR publication will fail since we don't have those collections e.g. MOD09GQ_<testid> in the CMR.

Reply:
I've updated the `addCollections` function to take a suffix to add to the collection name. I do get new collection configs on S3 and DynamoDB entries added in. You are right about the CMR problem, I'll think about how to work around that.

Reply:
Updated our `test-processing` package to remove substrings starting with `_test`